qa/suites/fs/basic_functional/tasks/volume-client/test.yaml

```diff
@@ -5,6 +5,10 @@ overrides:
       global:
         ms type: simple
 
+overrides:
```

Comment:
I don't think we need two `overrides:` in the same yaml.
Following can go under the existing `ceph:` task here https://github.com/ceph/ceph/pull/27109/commits/d839b07203706c647809d72e1bf672100bcd78ba#diff-3087565552d6bf9cfcd45fcb15e0acc7R3

`log-whitelist:`
￼       `- MON_DOWN`